espressostreamer/espresso_streamer.go

```diff
@@ -413,7 +434,13 @@ func (s *EspressoStreamer) Start(ctxIn context.Context) error {
 		} else {
 			log.Debug("Now processing hotshot block", "block number", s.nextHotshotBlockNum)
 		}
-		err := s.QueueMessagesFromHotshot(ctx, s.parseEspressoTransaction)
+		var err error
+		if s.isDecentralizedTimeboost {
```

Comment:
Instead of doing this, as I mentioned before it should be determined by espresso payload whether the given transaction is Timeboosted or not ( basically it should not be a config value)

We should discuss more as a team how we should enable this and not break compatibility with existing version

Theo has wanted to add headers to Espresso transactions